src/components/bottom-sheet/AnimatedBottomSheet.tsx

```diff
@@ -47,7 +47,7 @@ const useStyles = StyleSheet.createThemeHook((theme) => ({
     width: '100%',
     position: 'absolute',
     bottom: 0,
-    borderTopLeftRadius: theme.border.radius.circle,
-    borderTopRightRadius: theme.border.radius.circle,
+    borderTopLeftRadius: theme.border.radius.large,
+    borderTopRightRadius: theme.border.radius.large,
```

Comment:
The value that previously was `circle` has been renamed to `large`, and `circle` has been set to `999`. So changing every occurrence of it to `large` should keep things the same as before.